src/Components/CentralContainer/VerticalHandle/VerticalHandle.js

```
@@ -0,0 +1,71 @@
+        e.preventDefault();
+        e.stopPropagation();
+
+        document.addEventListener("mousemove", handleMouseMove);
+        document.addEventListener("mouseup", handleMouseUp);
+
+        downValueY.current = e.clientY;
+        topHeight.current = topDiv.current.getBoundingClientRect().height;
+        bottomHeight.current = bottomDiv.current.getBoundingClientRect().height;
+
+        handleRef.current.classList.add("handle-active");
+    }, [handleMouseMove, handleMouseUp]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)